addOns/retest/src/test/java/org/zaproxy/addon/retest/RetestPlanGeneratorUnitTest.java

```diff
@@ -43,13 +45,16 @@
 import org.zaproxy.addon.pscan.automation.jobs.PassiveScanWaitJob;
 import org.zaproxy.zap.network.HttpRequestBody;
 import org.zaproxy.zap.testutils.TestUtils;
+import org.zaproxy.zap.utils.I18N;
 
 class RetestPlanGeneratorUnitTest extends TestUtils {
 
     private static AutomationPlan retestPlan;
 
     @BeforeAll
     static void init() {
+        Constant.messages = new I18N(Locale.ENGLISH);
```

Comment:
I'd suggest doing in its own commit to stand it out.

Reply:
This was needed because of the change: https://github.com/zaproxy/zap-extensions/pull/5811/files#diff-0427963c8f73d14d2cf7381be4ed564f3b24fd7b8297358a9e63f5db66d24c41R487-R488. An NPE was being caused when calls to `JobUtils#strengthToI18n`and `JobUtils#strengthToI18n` were made.

The fix for the earlier failure was the change in `ExtensionOastUnitTests`. I'll move that out to a different commit.